pkg/kubelet/kubelet_pods.go

```diff
@@ -917,7 +917,7 @@ func (kl *Kubelet) PodResourcesAreReclaimed(pod *v1.Pod, status v1.PodStatus) bo
 		klog.V(3).Infof("Pod %q is terminated, but some containers have not been cleaned up: %s", format.Pod(pod), statusStr)
 		return false
 	}
-	if kl.podVolumesExist(pod.UID) && !kl.keepTerminatedPodVolumes {
+	if kl.podVolumesExist(pod.UID, false) && !kl.keepTerminatedPodVolumes {
```

Comment:
Nit: Can we do
```
checkVolumePaths := false
kl.podVolumesExist(pod.UID, checkVolumePaths)
```

otherwise, it can be difficult to know what the `false` means here.

Reply:
I added parameter name as a comment for `podVolumesExist` call. Thanks!

Reply:
Thanks for working with me on this small nit :)

This may just be my opinion, but I feel like the comment offers less advantages than the variable name (i.e. is less explicit). Are there advantages to using a comment over a variable that I'm missing?

Reply:
My IDE(goland) uses similar approach to show the parameter names of the function variables and I see similar usages in kubernetes code base, using an additional variable seems little noisy to me, so I choose adding comment.

But I agree, using variable can be more explicit, I'm neutral, I'll add additional variable instead of comment.

Reply:
Fixed

Reply:
Ah gotcha - maybe part of this is that I'm using vim haha :) Anyway, thanks!+

Reply:
Will let @jsafrane or someone else on sig-storage sign off :)